travis/apis.py

```diff
@@ -64,12 +64,9 @@ def _ssh_keyscan(self):
         The question is like to:
             'Are you sure you want to continue connecting (yes/no)?'"""
         cmd = ['ssh-keyscan', '-p']
-        match = re.search(r'@(?P<port_host>[^/]+)', self.ssh)
+        match = re.search(r'@([^/]+(?=:|/]))?(?::)?([^/]+)?/', self.ssh)
```

Comment:
@moylop260 
Good news! 
I found a regular expression to avoid a split function and another if statements